src/main/default/classes/TriggerConfigService.cls

```diff
@@ -1,12 +1,19 @@
 global class TriggerConfigService {
-    global static Map<String, ObjectConfig> configMetadata {
-        get {
-            if (configMetadata == null) {
-                configMetadata = buildConfigMap();
-            }
-            return configMetadata;
+    @TestVisible
+    private static Map<String, ObjectConfig> configMetadata;
+
+    /**
+     * Given the name of an SObject, returns the Config.  If one doesn't exist, returns a default config
+     */
```

Comment:
Refactored this to a method so it was easier to return a default value if one doesn't exist (so we don't have to do null checks everywhere)